src/js/index.js

```diff
@@ -11,30 +11,32 @@ var model = require('./model');
 model.init();
 
 // Initialize components
-var Sidebars = require('./components');
+var ui = require('./components');
 
 var g = model.Scene.child('marks.group'),
+    g2 = g.child('marks.group'),
+    g3 = g2.child('marks.group'),
```

Comment:
Correct, I added some nested groups when creating the primitives in `js/index.js`. Nesting adds a notable amount of complexity to the tree traversals involved in selection, so inasmuch as all of our homemade marks are for debugging/testing purposes, it felt appropriate to add some new levels to the hierarchy. If you object to them I can take them out but I have found them to be useful for smoke testing.